demisto_sdk/commands/lint/linter.py

```diff
@@ -321,6 +332,40 @@ def _run_flake8(self, py_num: float, lint_files: List[Path]) -> Tuple[int, str]:
 
         return SUCCESS, ""
 
+    def _run_xsoar_linter(self, py_num: float, lint_files: List[Path]) -> Tuple[int, str]:
+        """ Runs Xsaor linter in pack dir
+
+        Args:
+            lint_files(List[Path]): file to perform lint
+
+        Returns:
+           int:  0 on successful else 1, errors
+           str: Xsoar linter errors
+        """
+        with pylint_plugin(self._pack_abs_dir):
+            log_prompt = f"{self._pack_name} - XSOAR Linter"
+            logger.info(f"{log_prompt} - Start")
+            stdout, stderr, exit_code = run_command_os(
+                command=build_xsoar_linter_command(lint_files, py_num, self._facts.get('support_level', 'base')),
+                cwd=self._pack_abs_dir)
+
+        if exit_code == WARNING:
+            logger.warning(f"{log_prompt} - Finished warnings found : {stdout}")
+            exit_code = SUCCESS
+        logger.debug(f"{log_prompt} - Finished exit-code: {exit_code}")
+        logger.debug(f"{log_prompt} - Finished stdout: {RL if stdout else ''}{stdout}")
+        logger.debug(f"{log_prompt} - Finished stderr: {RL if stderr else ''}{stderr}")
+        if exit_code:
+            logger.info(f"{log_prompt}- Finished errors found")
+            if stderr:
+                return FAIL, stdout
```

Comment:
```suggestion
            if stderr:
                return FAIL, stderr
```

Reply:
it is intended to stay stdout as the actual message from pylint is returned in stdout.

Reply:
so please look at line 360-363, you are returning the same regardless of the condition.